src/pages/SwapList/SwapRow/SwapRow.tsx

```diff
@@ -92,6 +93,8 @@ function SwapRow({ swap, history, reload }: SwapRowProps) {
     link.rel.includes("human-protocol-spec")
   );
 
+  /* TODO: Force ellipsis on swap_id so that it doesn't take up
```

Comment:
📣 I trust you can do that before we merge :)

Reply:
I did a thing!